toolkits/landcover/test/int/landsat8.int.test.js

```diff
@@ -15,26 +15,44 @@
  * limitations under the License.
  */
 
-var assert = require('assert');
 var lct = require('../../api.js');
+var TestImage = require('../helpers/test-image.js');
 
 withEarthEngine('Landsat8', function() {
-  it('Filter and mask SR data', function(done) {
-    var point = ee.Geometry.Point(36.17819452553215, -4.826158059877228);
-    var image = lct.Landsat8('SR')
-                    .filterDate('2018-01-01', '2018-04-01')
-                    .filterBounds(point)
-                    .fmaskCloudsAndShadows()
-                    .getImageCollection()
-                    .mosaic();
+  fit('fmaskCloudsAndShadows() masks cloudy pixels', function(done) {
+    var clearPixel = TestImage.create({pixel_qa: 0, B4: 12});
+    var cloudyPixel =
+        TestImage.create({pixel_qa: lct.Landsat8.CLOUD_BIT_MASK, B4: 100});
+    var testCollection = ee.ImageCollection([clearPixel, cloudyPixel]);
+    var l8 = lct.Landsat8(testCollection);
 
-    // Cloudly pixel be masked out.
-    // TODO(gorelick): What would be a better way to test this?
-    var redValue = image.select('B4')
-                       .reduceRegion(ee.Reducer.first(), point, 10)
-                       .get('B4');
-    redValue.evaluate(function(actual) {
-      assert.equal(actual, null);
+    var maskedL8 = l8.fmaskCloudsAndShadows();
+
+    // Verify cloud-free pixel was returned.
+    var image = maskedL8.getImageCollection().mosaic();
+    var value = TestImage.reduceConstant(image).get('B4');
+    value.evaluate(function(actual, error) {
+      expect(error).toBeUndefined();
+      expect(actual).toEqual(12);
+      done();
+    });
+  });
+
+  fit('fmaskCloudsAndShadows() masks shadowy pixels', function(done) {
+    var clearPixel = TestImage.create({pixel_qa: 0, B4: 12});
```

Comment:
As these band values are capricious, maybe use some constants in the construction and test.
B4: CLEAR_PIXEL_VALUE, B4: CLOUDY_PIXEL_VALUE